packages/web/src/pages/admin.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { PageContainer } from '../components/layout/PageContainer';
+
+const AdminDashboard: NextPage = () => (
+  <PageContainer pageTitle={'Admin Dashboard'} heading={'Admin Dashboard'}></PageContainer>
```

Comment:
This is how you can display the page spinner:
```suggestion
  <PageContainer pageTitle={'Admin Dashboard'} heading={'Admin Dashboard'} loading={ true }></PageContainer>
```